frontend/src/components/review/CommentsPanel.tsx

```diff
@@ -13,19 +13,44 @@ export type CommentPanelProps = {
 
 const CommentsPanel = ({ storyTranslationId }: CommentPanelProps) => {
   const [comments, setComments] = useState<CommentResponse[]>([]);
-  const commentsQuery = buildCommentsQuery(storyTranslationId);
+  const [filterIndex, setFilterIndex] = useState(0);
+
+  const filterOptions = ["Show All", "Show Resolved", "Show Unresolved"];
+  const filterToResolved = [null, true, false];
```

Comment:
nitpick: Instead of using two arrays, could you use "null, true, false" as the state variable and have a dictionary to access what is currently in `filterOptions`? And `.keys()` and `.values()` if necessary
- I'm also not sure if this is more clear, but it prevents having to use additional logic to map between the two?

```typescript
const [filterResolvedOnly, setFilterResolvedOnly] = setState<Boolean | null>(null);
...
{
  null: "Show All",
  true: "Show Resolved",
  false: "Show Unresolved"
}
```

Reply:
`filterIndex` works nicely with data available in `event.target`. Going with the map would require a reverse map (eg "show all" -> null) to go from anything in `event.target` back to a valid `resolved` value. IMO the two options here are equally annoying LOL